citizen/citizen.c

```diff
@@ -9,129 +9,151 @@
 int counter = 0;
 
 
-// compare two Citizen objects based on the specified sort type.
-// Returns the result of the comparison.
-int cmpCtz(const Citizen a, const Citizen b, Type sort) {
+/*
+ * Compare two Citizen objects based on the specified sort type.
+ * Returns the result of the comparison.
+ */
+int compareCitizen(const Citizen a, const Citizen b, Type sort)
```

Comment:
maybe this can be turn into Citizen first_person, Citizen second_person or make the variable name descriptive

Reply:
Renamed it to citizen_a. Job is done. Thanks.